overlord/snapstate/snapstate.go

```diff
@@ -153,7 +154,7 @@ func doInstall(s *state.State, snapst *SnapState, ss *SnapSetup) (*state.TaskSet
 	return state.NewTaskSet(tasks...), nil
 }
 
-func checkChangeConflict(s *state.State, snapName string) error {
+func checkChangeConflict(s *state.State, snapst *SnapState, snapName string) error {
```

Comment:
I would switch the order of params, is a bit strange the nil in the middle in many cases